pages/common/mongo.md

```diff
@@ -0,0 +1,19 @@
+# mongo
+
+> MongoDB shell client.
```

Comment:
Yea I had considered this. I think shell is a better choice here. Mainly because this actually creates a javascript shell environment wherein you can execute JS code apart from your usual DB queries. Even the man pages and official docs refer to it as a "shell" client.